src/hotspot/cpu/aarch64/interpreterRT_aarch64.cpp

```diff
@@ -306,7 +400,7 @@ class SlowSignatureHandler
       *_int_args++ = from_obj;
       _num_int_args++;
     } else {
-      *_to++ = from_obj;
+      store_and_inc(_to, from_obj, NativeStack::longSpace);
       _num_int_args++;
```

Comment:
`pass_byte()` and  `pass_short()` use only one `_num_int_args++;` after the `if else` but other methods use it 2 of them inside `if else`

We should be consistent.